### ISSUE TITLE
ORKActiveStepTimer delay in calling _handler() first time after app launch

### ISSUE THREAD

**Issue by rsanchezsaez** [ResearchKit/ResearchKit#155]
`ORKActiveStepTimer` seems to be misbehaving the first time it calls `_handler()` after app launch when connected to Xcode for debugging. 100% reproducible on an *iPhone 6* running *iOS 8.3*.

Not sure if this is a valid issue or a debugging artifact.

###### Steps to reproduce 
1. Freshly launch ORKTest.
2. Open the Fitness example.
3. Start the activity and observe the '*Starting activity in*' timer.

###### Observed result
1. Timer shows 5.
2. After two seconds, timer shows 4 and 3 in quick succession.
3. Timer works correctly for the remaining countdown.

###### Expected result
1. Timer shows 5.
2. Timer shows 4 after 1 second.
3. Timer works correctly for all the countdown.

###### Notes
- Issue doesn't occur if you launch the Fitness activity again without quitting the app.
- Issue doesn't occur if you launch the app without attaching from Xcode for debugging.
- Issue doesn't occur on the the simulator.

**Comment by jwe-apple** [comment]
Yep aware of this. As you say, only ever happens attached to the debugger.
